carsus/data/cmfgen_config.yml

```diff
@@ -107,3 +107,57 @@ atom:
         osc: s2_osc
         pho:
           - phot_sm_3000
+
```

Comment:
Line 78 add Co II and Co I

Reply:
Also N II in line 86

Reply:
I'm afraid I was not able to find file names for Co I in [readhillierdata.py](https://github.com/artis-mcrt/artisatomic/blob/8bb6e180509e7962c8430725b7a7d5ba54df4af9/artisatomic/readhillierdata.py#L160-L166) or in the CMFGEN Dataset.